Feature/tes

Ktayama opened the pull request.

Ktayama committed:
プルリクエストtest

Ktayama committed:
テスト２削除

Ktayama committed:
List.Sevlet更新

Ktayama committed:
Testサーブレットの確認お願いします。

yoshizaki-takeshi commented on tes/src/main/java/Test_result.java:
```diff
@@ -0,0 +1,102 @@
+			for(int j = 0;j< list.size(); j++)
+			{
+				
+				if(list.get(j).getAnswer().equals(corrct_answer[i]))
```
ここの条件、念のためこうした方が安全だと思います。
```
if(list.get(j) != null && list.get(j).getAnswer().equals(corrct_answer[i]))
````
ｊ番目の値が空だった場合、getAnswer()メソッドでエラー落ちする可能性も出てくるので…

Ktayama committed:
test処理追加

yoshizaki-takeshi commented on tes/src/main/webapp/test_result.jsp:
```diff
@@ -0,0 +1,15 @@
+</head>
+<body>
+
+<%String[] questions_id = (String[])request.getAttribute("Question_cnt");%>
```
Question_cntはint型でセットしてるのでStringの配列でなくint型で受け取るべきと思います。

yoshizaki-takeshi commented on tes/src/main/webapp/test_result.jsp:
```diff
@@ -0,0 +1,15 @@
+<body>
+
+<%String[] questions_id = (String[])request.getAttribute("Question_cnt");%>
+<%String[]  = (String[])request.getAttribute("Question_cnt");%>
```
途中なのかもしれませんが、変数名が書かれていませんね。

yoshizaki-takeshi commented on tes/src/main/webapp/test.jsp:
```diff
@@ -0,0 +1,55 @@
+
+<%ArrayList<QuestionsBean> Question_list = (ArrayList<QuestionsBean>)request.getAttribute("Qusertions_list");%>
+<!-- List.javaからのパラメータをArrayList<Correct_answersBaen>型の変数に取得する -->
+<%ArrayList<Correct_answersBean> Correct_answer_list = (ArrayList<Correct_answersBean>)request.getAttribute("Correct_anser_list");%>
```
悪さはしないと思いますが使用してないので消しちゃっても良いですよ。

Ktayama committed:
test_result.jspの確認お願いします。

yoshizaki-takeshi commented on tes/src/main/webapp/test_result.jsp:
```diff
@@ -0,0 +1,34 @@
+</head>
+<body>
+
+<%int user = (int)request.getAttribute("GetName");%>
```
GetNameはString型になると思います。

Ktayama committed:
経過報告HistoriesDaoがエラーが出てます。
それ以外は、エラーは出ていません

Ktayama committed:
Historiesを変更してみました。
エラーがはいてしまったので、お時間ありましたら確認おねがいします。
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'where user_id = '11'' at line 1

Ktayama committed:
JUnit単体テストをしましたので、お時間ありましたら確認お願いします。

Ktayama committed:
更新しましたのでご確認よろしくお願いします。

Ktayama committed:
jspの変更＆追加しましたので確認よろしくお願いします。

Ktayama committed:
HistoriesDao修正版です。ご確認よろしくお願いします。

Ktayama committed:
CSS用に変更したJSPになります。
ご確認の程宜しくお願いいたします。

Ktayama committed:
CSSの追加処理になります。
ご確認の程よろしくお願いいたします。

Ktayama committed:
変更、追加しました。
ご確認よろしくお願いします。

Ktayama committed:
変更・追加しましたサーブレットです。
ご確認よろしくお願いします。

Ktayama committed:
編集のエラーが治りましたので、確認の程宜しくお願いいたします。
